### ISSUE TITLE
Add an option to open links in private window

### ISSUE THREAD

**Issue by VodnikAngus** [sonnyp/Junction#58]
It would be great to add an option to open links in private window

**Comment by sonnyp** [comment]
It's already there - right click on the browser icon -> open in private window

**Comment by VodnikAngus** [comment]
Right click menu shows up but when I click either option it doesn't work, just quits the app. I tried both Firefox and Chromium

**Comment by sonnyp** [comment]
1. Please share the content of menu -> About -> System
2. Reproduce the problem after opening Juncton in the terminal to share the logs

```sh
flatpak run re.sonny.Junction https://github.com
```

**Comment by VodnikAngus** [comment]
``menu`` -> ``About`` -> ``System``:
```
Junction:
version 1.4.0
programInvocationName /app/bin/re.sonny.Junction
argv --gapplication-service
cwd /home/petar
datadir /app/share

Powered by:
GJS 1.70.0
libadwaita 1.0.0
GTK 4.4.1
GLib 2.70.2
flatpak 1.12.3

Environment:
OS Fedora Linux 35 (Workstation Edition)
$XDG_DATA_DIRS /app/share:/usr/share:/usr/share/runtime/share:/run/host/user-share:/run/host/share:/run/host/usr/share:/var/lib/snapd/desktop:/var/lib/flatpak/exports/share:/home/petar/.local/share/flatpak/exports/share
$PATH /app/bin:/usr/bin
$FLATPAK_ID re.sonny.Junction
$XDG_CURRENT_DESKTOP GNOME
$XDG_SESSION_TYPE x11
```

``flatpak run re.sonny.Junction https://github.com``:
```
$ flatpak run re.sonny.Junction https://github.com
sh: line 1: exec: firefox: not found
```

Probably something wrong with Flatpak permissions

**Comment by sonnyp** [comment]
Fixed in Junction 1.5.0 available on flathub.

**Comment by Zzombiee2361** [comment]
Hi, I don't see any menu on right click. I have firefox and chrome, right clicking either do nothing

<details>
  <summary>Here's my system info</summary>

  ```
  Junction:
  version 1.5.0
  programInvocationName /app/bin/re.sonny.Junction
  argv --gapplication-service
  cwd /home/daffa
  datadir /app/share
  
  Powered by:
  GJS 1.70.2
  libadwaita 1.0.0
  GTK 4.4.2
  GLib 2.70.5
  flatpak 1.12.7
  
  Environment:
  OS Ubuntu 22.04 LTS (Jammy Jellyfish)
  $XDG_DATA_DIRS /app/share:/usr/share:/usr/share/runtime/share:/run/host/user-share:/run/host/share:/run/host/usr/share:/var/lib/snapd/desktop:/var/lib/flatpak/exports/share:/home/daffa/.local/share/flatpak/exports/share
  $PATH /app/bin:/usr/bin
  $FLATPAK_ID re.sonny.Junction
  $XDG_CURRENT_DESKTOP KDE
  $XDG_SESSION_TYPE x11
  ```
</details>

**Comment by spi43984** [comment]
Am on version 1.7 - don't see a menu on right click either.

**Comment by spi43984** [comment]
got it resolved, pls see #107
